model/common/tests/grid_tests/utils.py

```diff
@@ -48,6 +51,9 @@ def resolve_file_from_gridfile_name(name: str) -> Path:
                 r02b04_global_data_file,
             )
         return gridfile
+    elif name == "0055_R02B05":
```

Comment:
this is a very specific name. I think we should define a string above with this name and then use it here since it may change in the future

Reply:
Not that it changes anything in terms of code functionality, but it is best practice

Reply:
Good catch, I think I added it to try out something and it should be wanted to remove it again.